org.opentosca.yamlconverter.main/src/main/java/org/opentosca/yamlconverter/main/Parser.java

```diff
@@ -21,11 +22,17 @@
 	private ServiceTemplate serviceTempl = null;
 	private Definitions definition = null;
 
+	// TODO: Is this map still needed?!
```

Comment:
@seeebiii Yes, this will be used in fillGetter(), where the inputs will be pasted into the XML